archive/common/src/main/scala/uk/ac/wellcome/platform/archive/common/progress/models/Callback.scala

```diff
@@ -18,27 +18,19 @@ case object Callback extends URIConverters {
     callbackUri.map(Callback(_))
   }
 
-  val pendingString = "pending"
-  val succeededString = "succeeded"
-  val failedString = "failed"
+  val processingString = "processing"
+  val successString = "success"
+  val failureString = "failure"
 
   case object Pending extends CallbackStatus {
-    override def toString: String = pendingString
+    override def toString: String = processingString
```

Comment:
Maybe rename the class to `Processing` to match the string?  Personally I prefer `pending` for the callback status, because processing leads me to think it is waiting for a response.